### DISCUSSION TITLE
Open for debate with the other side, in case they accept? UPDATE: Other side banned me for purposing the same.

### DISCUSSION THREAD

**Discussion by waltercool** [rms-support-letter/rms-support-letter.github.io#2555] [General]
Understanding the idea of keeping two different "open letters" and observe who cancel each other, is there a possibility from this "letter" to just discuss this matter  with the other one?

I do acknowledge this is a counter-cancel culture, but I feel like no discussion means no solution.

By the way, I do recognize keeping open discussions is lot better than standing against any response (like anti-rms letter), so thanks for keeping this channel at least open.

**Comment by nukeop** [comment]
Anyone who tried to discuss anything contrary to the letter contents in the other repo got banned. They're not interested in discussion, it's about tribal allegiance.

**Reply by waltercool** [reply]
I know. Just opened the same argument at "the other side" at https://github.com/rms-open-letter/rms-open-letter.github.io/issues/2259 in order to make them understand this strategy won't help free software, won't help foundations supporting the open letter (EFF / Mozilla / Tor / Debian) or anyone who use free software.

If they close and remove that issue "as well", would be very sad for them, trying to force remove someone without reasoning isn't democratic, isn't healthy, and definitively won't resolve their complains.

**Reply by purplesyringa** [reply]
QED.

![image](https://user-images.githubusercontent.com/16370781/112614259-1b274700-8e32-11eb-9413-cd257fc6e497.png)

**Reply by thatHguy** [reply]
They seem using the same approach that the press and the mainstream media uses on important issues where whey have their own narrative to push, and there has been a **whole lot** of that since the end of 2019. They'll only hear and parrot their own propaganda and will shut down any other discussions. This is a smear campaign with an agenda behind it, without a doubt.

**Comment by AnimusPEXUS** [comment]
how do you see the compromise? : half of Stallman to left and half to right?

**Reply by waltercool** [reply]
Nope, I'm pro Stallman on this, while I'm not GNU supporter.

What they are doing is wrong, a coup never ends well. My concern here is what's next. Making two open letters and no discussing it's wrong in multiple layers, if we don't find a way to resolve this with discussion, no one wins.

Specially now, when some charitable foundations are joining this coup movement, or crazy people purposing cancelling FSF as well https://github.com/rms-open-letter/rms-open-letter.github.io/issues/454.

The only way to resolve that is discussing it, finding common grounds... like RMS not resigning, but maybe giving a public apologize for whatever they claim... like an idea. Thing is, those guys aren't even intending to start a discussion, which is more harmful for them rather for "us" who disagree with the cancel idea.

**Reply by AnimusPEXUS** [reply]
why should him apologize if he doesn't feel he done (or said) wrong? - this is nonsense.

As I see this situation: this is just another big tech attack on freedoms and on freedom of speech in particular. Stallman, FSF  and other free software individuals have been under attacks in the past and this one attack is just another attempt to shutoff the leader of greatest anti-slavery company. So nothing new here. Those who want to separate - freedom is theirs. But Stallman should remain untouched.

**Reply by waltercool** [reply]
I agree, but Stallman is a public figure. If there are "organizations" like the whole Gnome, Debian or Mozilla team backing this coup to himself AND FSF director members, it means they have a strong hate feeling about it.

At any democracy is not okay to demand the head of people elected, so the best solution is to reason with the "other side". Now... the problem is the non-debatable position of themselves... they went full Robespierre. 

If they consider Stallman said or did something it hurt their feelings, so maybe Stallman should apologize about those claims... even if he haven't said/did those ones. But resign....? No.

Current problem is entirely political, isn't even software or philosophical related, so the solutions must be done politically. Resignation of anyone who was elected is completely nonsense and r-tarded. Coups never have good endings.

**Comment by Jan200101** [comment]
as it stands I am open to discussion and anyone that is interested in talking to me can find me
- here
- on Discord (Jan#6008)
- any other medium you manage to contact me on

**Comment by waltercool** [comment]
Update to this.

They just closed and deleted my ticket, they also banned me, even if was trying to find common grounds instead of blaming wars. So they definitively don't want to discuss anything, they are just an angry mob.

So sad they don't want to even discuss their points.

This is a copy of my original ticket before they delete it: https://archive.is/RMlmV

**Comment by HBelusca** [comment]
Those who are banning lustily just have the mentality of 5 years old from school playgrounds.

**Comment by Aspie96** [comment]
On their letter I respectfully (NO, I was NOT trolling) asked them to clarify Stallmans quote and include the following sentence of the email.

They banned me from the org.

**Reply by Aspie96** [reply]
Ah, and for those who disagree with me: I am open to discussion!

Here and on twitter

**Reply by waltercool** [reply]
They don't intend to discuss anything, that's the problem.

They want to impose it. Otherwise they wouldn't be deleting and banning

**Reply by Aspie96** [reply]
I do want to discuss, though.
So I put my contact info.

Whether to discuss or not is their choice

**Reply by Aspie96** [reply]
I wonder how many are aware of the situation.

Some of those people are people I appreciate. I think many signed the letter and are unaware of how it's being run

**Comment by cmpunches** [comment]
Dude they have a funded PR team running that effort.  They're not going to talk to any of us because they've broken the law and they're mitigating, distracting, and generally causing unrest with viral shadow campaigns from here.  Their intent is to create organizational pressure.  Their effort was derailed by the support letter for RMS.  Now they're mad and the PR team is on retainer.  Ride this out.
